loader/status.go

```diff
@@ -63,7 +63,7 @@ func (l *Loader) PrintStatus(ctx context.Context) {
 
 		finishedSize := l.finishedDataSize.Get()
 		totalSize := l.totalDataSize.Get()
-		log.Infof("[loader] finished_bytes = %d, total_bytes = GetAllRestoringFiles%d, progress = %s", finishedSize, totalSize, percent(finishedSize, totalSize))
+		log.Infof("[loader] finished_bytes = %d, total_bytes = GetAllRestoringFiles = %d, progress = %s", finishedSize, totalSize, percent(finishedSize, totalSize))
```

Comment:
```suggestion
		# get filesCount from l.db2Tables maybe
		log.Infof("[loader] finished_bytes = %d, total_bytes = %d, GetAllRestoringFiles = %d, progress = %s", finishedSize, totalSize, filesCount, percent(finishedSize, totalSize))
```
here we miss the format string for `total_bytes` and real count for `GetAllRestoringFiles`. Maybe we can get `GetAllRestoringFiles` from `l.db2Tables` or adding another counter.